test/configs/zombieTanssi.json

```diff
@@ -64,31 +64,31 @@
           {
             "name": "Collator2002-01",
             "command": "../target/release/tanssi-node",
-            "args": ["-- --tmp --prometheus-external --rpc-cors all --unsafe-rpc-external --rpc-methods unsafe --para-id 2002"]
+            "args": ["-- --tmp --para-id 2002"]
```

Comment:
This is not needed, the `--para-id` arg should only be used when running a full node, and this is a collator.

```suggestion
```